bindings/python/test/utils.py

```diff
@@ -145,7 +145,10 @@ def setUp(self):
 
     def assertType(self, obj1, arrow_type):
         if isinstance(obj1, pyarrow.ChunkedArray):
-            if "storage_type" in dir(arrow_type):
+            if "storage_type" in dir(arrow_type) and arrow_type not in (
+                ObjectIdType(),
```

Comment:
Perhaps we can check for an extension type here so we don't have to manually update?